frontend/reducers/modal_reducer.js

```diff
@@ -0,0 +1,19 @@
+import * as actions from '../consts/action-types';
+
+const defaultState = {
+  active: false,
```

Comment:
maybe prefer `active` to refer to a specific modal. assumption is that only one modal can be rendered in the app?